src/routes/quickstarts/cypress/setup.mdx

```diff
@@ -0,0 +1,172 @@
+# Mock Network Requests in Cypress with Mirage
+
+Use your Mirage server to test your application under different server scenarios using [Cypress](https://www.cypress.io/).
+
+> This is a quickstart guide for people already using Cypress in their apps.
+
+## Step 1: Install Mirage
+
+First, make sure you have Mirage installed:
+
+```bash
+# Using npm
+npm install --save-dev miragejs
+
+# Using Yarn
+yarn add --dev miragejs
+```
+
+## Step 2: Define your server
+
+Create a new `src/server.js` file and define your mock server.
+
+Here's a basic example:
+
+```js
+// src/server.js
+import { Server, Model } from "miragejs"
+
+export function makeServer({ environment = "development" } = {}) {
+  let server = new Server({
+    environment,
+
+    models: {
+      user: Model,
+    },
+
+    seeds(server) {
+      server.create("user", { name: "Bob" })
+      server.create("user", { name: "Alice" })
+    },
+
+    routes() {
+      this.namespace = "api"
+
+      this.get("/users", schema => {
+        return schema.users.all()
+      })
+    },
+  })
+
+  return server
+}
+```
+
+## Step 3: Configure Cypress to define a proxy function for your app's API requests
+
+Add the following code to your `cypress/support/index.js` file:
+
+```js
+// cypress/support/index.js
+Cypress.on("window:before:load", win => {
+  win.handleFromCypress = function(request) {
+    return fetch(request.url, {
+      method: request.method,
+      body: request.requestBody,
+    }).then(res => {
+      if (res.headers.map["content-type"] === "application/json") {
+        return res.json()
+      } else {
+        return ""
+      }
+    })
+  }
+})
+```
+
+This code defines a `handleFromCypress` function on your application's `window` object. In the next step, we'll configure your app to call this function whenever it makes a network request and Cypress is running.
+
+## Step 4: Proxy your app's network requests
+
+In your app's bootstrapping file, start Mirage if `window.Cypress` is true.
```

Comment:
"In your app's bootstrapping file, use Mirage to proxy your app's API requests to the `handleFromCypress` function when Cypress is running."